files/build_templates/init_cfg.json.j2

```diff
@@ -82,11 +82,17 @@
             "auto_restart": "{{autorestart}}",
             "support_syslog_rate_limit" : "true",
 {# Set check_up_status to true here when app readiness will be marked in state db  #}
+{%- if feature in ["swss", "syncd", "pmon"] %}
+            "check_up_status" : "true",
```

Comment:
Please do not change image config generation behavior. We will rely on external service to generate golden config, and ask sonic to `config reload -f golden_config.json` to consume them.

Reply:
@wen587 to review.